spec/app/ics-028-cross-chain-validation/system_model_and_properties.md

```diff
@@ -169,7 +169,7 @@ As a result, it is possible for multiple VSCs to be received by a consumer chain
 i.e., the validator updates within the VSCs are being *aggregated* by keeping only the latest update per validator. 
 As a consequence, some validator sets on the provider chain are not existing on all consumer chains. 
 In other words, the validator sets on each consumer chain form a *subsequence* of the validator sets on the provider chain. 
-Nonetheless, as a **requirement of CCV**, *all the validator updates on the provider chain MUST be included in the sequence of validator sets on all consumer chains*.
+Nonetheless, as a **requirement of CCV**, *all the updates to validator power on the provider chain MUST be included in the sequence of updates to validator power on all consumer chains*.
```

Comment:
I would keep it as it is. The reason is twofold. 

First, "validator updates" is described above, i.e., 
> validator updates (i.e., changes in the validators voting power)

Thus, I find that the phrase  "all the validator updates on the provider chain" does not need further clarification. 

Second, the sequence of validator sets on a consumer chain is the result of applying a sequence of VSCs, which contains a sequence of validator updates. This is described in the following lines
> the sequence of blocks produces a sequence of validator updates and a sequence of validator sets. Furthermore, the sequence of validator updates on the provider chain results in a sequence of VSCs to all consumer chains. Ideally, this sequence of VSCs is applied by every consumer chain, resulting in a sequence of validator sets identical to the one on the provider chain

By saying that a validator update U1 must be "included" in a validator set, I mean that the results of either applying U1 or a subsequent update U2 that includes U1 must be visible in the validator set. I agree that "included" may not be the best word here, but I couldn't come with a better one.

Reply:
Hmm, ok, fair enough. It was good to discuss anyway. Thanks.